iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergTimeTravel.java

```diff
@@ -49,8 +49,8 @@ public void testSelectAsOfTimestamp() throws IOException, InterruptedException {
     Assert.assertEquals(4, rows.size());
 
     AssertHelpers.assertThrows("should throw exception", IllegalArgumentException.class,
-        "Cannot find a snapshot older than 1970-01-01 00:00:00", () -> {
-          shell.executeStatement("SELECT * FROM customers FOR SYSTEM_TIME AS OF '1970-01-01 00:00:00'");
+        "java.lang.IllegalArgumentException: Cannot find a snapshot older than 1970-01-01", () -> {
+        shell.executeStatement("SELECT * FROM customers FOR SYSTEM_TIME AS OF '1970-01-01 00:00:00.000'");
```

Comment:
Out of curiosity: does it not work without the `.000` part in the SQL statement?

Reply:
Yup it does, this is probably just a leftover from something, I'll remove it.